frontend/server/src/DAO/Contests.php

```diff
@@ -634,6 +634,120 @@ final public static function getContestsParticipating(
+            \OmegaUp\DAO\Enum\ActiveStatus::ACTIVE
+        );
+        $futureCondition = \OmegaUp\DAO\Enum\ActiveStatus::sql(
+            \OmegaUp\DAO\Enum\ActiveStatus::FUTURE
+        );
+        $columns = \OmegaUp\DAO\Contests::$getContestsColumns;
+
+        $select = "SELECT
+                        $columns,
+                        p.scoreboard_url,
+                        p.scoreboard_url_admin,
+                        COUNT(contestants.identity_id) AS contestants,
+                        ANY_VALUE(organizer.username) AS organizer";
+
+        $sql = "
+            FROM
+                (SELECT
+                    pi.problemset_id
+                FROM
+                    Problemset_Identities pi
+                WHERE
+                    pi.identity_id = ?
+                UNION DISTINCT
+                SELECT
+                    p.problemset_id
+                FROM
+                    Groups_Identities gi
+                INNER JOIN
+                    Group_Roles gr ON gi.group_id = gr.group_id
+                INNER JOIN
+                    Problemsets p ON gr.acl_id = p.acl_id
+                WHERE
+                    gi.identity_id = ? AND gr.role_id = ?
+                UNION DISTINCT
+                SELECT
+                    p.problemset_id
+                FROM
+                    Teams t
+                INNER JOIN
+                    Teams_Group_Roles tgr ON t.team_group_id = tgr.team_group_id
+                INNER JOIN
+                    Problemsets p ON tgr.acl_id = p.acl_id
+                WHERE
+                    t.identity_id = ? AND tgr.role_id = ?
+                ) pps
+            INNER JOIN
+                Contests
+            ON
+                Contests.problemset_id = pps.problemset_id
+            INNER JOIN
+                Problemsets p
+            ON
+                p.problemset_id = Contests.problemset_id
+            LEFT JOIN
+                Problemset_Identities AS contestants
+            ON
+                Contests.problemset_id = contestants.problemset_id
+            INNER JOIN
+                ACLs AS a
+            ON
+                Contests.acl_id = a.acl_id
+            INNER JOIN
+                Identities AS organizer
+            ON
+                a.owner_id = organizer.user_id
+            WHERE
+                ($activeCondition OR
+                $futureCondition) AND
+                archived = 0
+            GROUP BY Contests.contest_id
+        ";
+        $params = [
+            // Direct participation
+            $identity->identity_id,
+            // Group participation
+            $identity->identity_id,
+            \OmegaUp\Authorization::CONTESTANT_ROLE,
+            // Team participation
+            $identity->identity_id,
+            \OmegaUp\Authorization::CONTESTANT_ROLE,
+        ];
+
+        $limits = '
+            ORDER BY
+                start_time,
+                finish_time;
```

Comment:
Acá si deberíamos mantener el `LIMIT` para garantizar que sólo se va a obtener un registro:
```suggestion
            ORDER BY
                start_time ASC,
                finish_time ASC
            LIMIT 1;
```

Además de indicar la forma en que se van a ordenar los registros para que sea más clara esta consulta